### PR TITLE
Add function to export quiz to TeX document of class examdesign

### PR DESCRIPTION
**Pull request by bapfeld**
It's a little rudimentary and I haven't tested it thoroughly, but it works for multiple choice questions with any number of correct answers.

### CODE REVIEW

File: R/export_quiz.R [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#' Export quiz as TeX
+#'
+#' @param quiz a data.frame created by import_quiz
+#' @param output_file the location to save the TeX document
+#' @param class_name name of class to be used on quiz
+#' @param quiz_name name of quiz to be used on quiz
+#' @param n_ver number of versions to create when TeX is compiled
+#' @return outputs a TeX file to output_file location
+#' @export
+export_quiz <- function(quiz, output_file, class_name, quiz_name, n_ver){
```
Comment (jabranham) [inline R/export_quiz.R]: Shouldn't `class_name` and `quiz_name` default to `""` and `n_ver` default to 1?

Comment (bapfeld) [reply R/export_quiz.R]: Yeah, I think that makes sense.

Comment (bapfeld) [reply R/export_quiz.R]: Actually, no. It needs characters in those fields or else LaTeX will throw an error when you try to compile.

Comment (jabranham) [reply R/export_quiz.R]: Hrm. Can we set the default to NULL and use an if statement to only set the names if the argument exists?

Comment (bapfeld) [reply R/export_quiz.R]: No, apparently examdesign requires those arguments

**Pull request merged**